ts/component/table-builder.ts

```diff
@@ -135,28 +138,28 @@ export class TableBuilder<ParentType, ChildType> {
     }
     this.baseFilterKey = container.getAttribute("data-detail-type");
     this.baseFilterValue = container.getAttribute("data-detail-value");
-    if (searchParams) {
-      searchParams.forEach((p) => {
-        // parse url search params and check for a matching key in table definition
-        var key = p.at(0);
-        var value = p.at(1);
-        if (
-          this.definition.filters?.find((f) => {
-            return f.title === key;
-          }) &&
-          key &&
-          value
-        ) {
-          // create accepted filter if there is a matching key in table definition
-          this.acceptedFilters.push(
-            new AcceptedFilter(key, key.toUpperCase(), value, () =>
-              this.reload()
-            )
-          );
-        }
+
+    if (filterParams) {
+      // create accepted filter if there is a matching key in table definition
+      filterParams.forEach((p) => {
+        this.definition.filters?.find((f) => {
```

Comment:
Could use `forEach` instead of `find` since you're not doing anything with the return value